src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+	<ThresholdFilter level="all" />
+	<Appenders>
+		<Console name="STDOUT" target="SYSTEM_OUT">
+			<PatternLayout
+				pattern="[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n" />
+		</Console>
+		<RollingFile name="RollingFile" fileName="${sys:logPath}/CC-LOG.log"
+			filePattern="${sys:logPath}/CC-LOG-%d{yyyy-MM-dd}-%i.log"
+			ImmediateFlush="true" AppendToFile="true">
```

Comment:
do we need `ImmediateFlush`? What does that do to performance? or is that an unknown?